doc/developer-guides/coding_guidelines.rst

```diff
@@ -3310,3 +3310,47 @@ The number of bytes in an object is implementation-defined, according to J.3.13
 item 2 in C99. For ACRN hypervisor, char is 1 byte, short is 2 bytes, int is 4
 bytes, long is 8 bytes, and long long is not used.
 
+Language Extensions
+*******************
+
+
+LE-01: Inline Assembly language in C Code is used
```

Comment:
Should these all say "may be used" (are you giving permission as part of the coding standard)?
Also, can some of these be combined, for example  put all the type attributes together, and the build-in types and functions together?

Reply:
Combination of these features is allowed.

Could you please provide more details about this question? Are you talking about the GCC manual?
`are you giving permission as part of the coding standard?`